src/manifolds/Rotations.jl

```diff
@@ -344,6 +344,33 @@ function Random.rand!(
     return pX
 end
 
+"""
+    _get_expdhalf(::Rotations, d)
+
+Calculate `exp(d / 2)`.
+"""
+_get_expdhalf(::Rotations, d) = exp(d / 2)
+"""
+    _get_expdhalf(::Rotations{TypeParameter{Tuple{3}}}, d)
+
+Calculate `exp(d / 2)` for the manifold of `Rotations(3)` based on the Rodrigues' rotation
+formula.
+"""
+function _get_expdhalf(::Rotations{TypeParameter{Tuple{3}}}, d)
+    θ = norm(d) / (2 * sqrt(2))
+    if θ ≈ 0
+        a = 1 - θ^2 / 6
+        b = θ / 2
+    else
```

Comment:
Also already in earlier code, I did not understand these lines too much. Here `θ` is nearly zero, maybe up to a small type dependent tolerance.

But then up to the same tolerance `a` is just 1 (actually with a much smaller tolerance) and `b` is rust 0, or in short this could (together with the `a/2` later just return `I + 0.5 .* d`?

Reply:
Hm, the mention of AD I do not understand, for me these lines look useless ;)

Reply:
Sure, but that is even no where mentioned here, so I can not guess that it is necessary for that

Reply:
You can always ask me if you need to know.